src/Process/Pool/Logger/Message/SerializableProcess/FromProcessModel/AwareTrait.php

```diff
@@ -8,47 +8,47 @@
 /** @codeCoverageIgnore */
 trait AwareTrait
 {
-    protected $NeighborhoodsKojoProcessPoolLoggerMessageProcessFromProcessModel;
+    protected $NeighborhoodsKojoProcessPoolLoggerMessageSerializableProcessFromProcessModel;
 
-    public function setProcessPoolLoggerMessageProcessFromProcessModel(
-        FromProcessModelInterface $ProcessPoolLoggerMessageProcessFromProcessModel
+    public function setProcessPoolLoggerMessageSerializableProcessFromProcessModel(
```

Comment:
Ah okay this address my comment [here](https://github.com/neighborhoods/kojo/pull/76#discussion_r327284183)

Just want to make sure again that all calls to this method in YAML files were updated as well